app/factory/Layer.js

```diff
@@ -240,10 +242,15 @@ Ext.define('CpsiMapview.factory.Layer', {
             // this within the function is bound to the vector source it's
             // called from.
             var timeFilters = this.get('timeFilters');
-
             if (!Ext.isEmpty(timeFilters)) {
                 allFilters = Ext.Array.merge(allFilters, timeFilters);
             }
+            var numericFilters = this.get('numericFilters'); // TODO docs
```

Comment:
Anything up to come since there are several `TODO` tags in the following lines? Or just a reminder for future tasks?

Reply:
I am not sure where I would document this. Do we have one place where the layer properties we use internally are documented? Likewise, for the initial JSON where is the one true documentation for the structure and allowed values?

Reply:
I'm have a drafy JSONSChema doc for the JSON config at http://w08-mapserver.compass.ie/mapview-config-docs/. I'll move it into this project in the future, At the moment the config is very flexible and driven more by the feature requirements than a config spec, so any changes to the config are fine.